src/main/java/org/opentripplanner/middleware/utils/HttpUtils.java

```diff
@@ -22,7 +22,7 @@
 
 public class HttpUtils {
     private static final Logger LOG = LoggerFactory.getLogger(HttpUtils.class);
-    public enum REQUEST_METHOD {GET, POST, DELETE}
+    public enum REQUEST_METHOD {GET, POST, PUT, DELETE} // FIXME: use sn existing library enum?
```

Comment:
This has been removed in the error-notifications PR (#111). Depending on which gets merged first we'll hit a conflict. Guess that's unavoidable though.

Reply:
It is. I'm just realizing I left a typo in the FIXME comment.

Reply:
I've removed the comment as it will be addressed when error-notifications PR is merged.